meson.build

```diff
@@ -754,6 +754,7 @@ if features['zimg']
         sources += files('test/repack.c',
                          'test/scale_zimg.c')
     endif
+    features += {'zimg-st428': zimg.version().version_compare('>= 3.0.5')}
```

Comment:
I don't know how zimg API change policy works but based on the public header only defining the major/minor API version, I think it would have to be a bump to `3.1.0` in order to expose the new enum member?

(In which case, we also don't need to do this check manually)

Reply:
https://github.com/sekrit-twc/zimg/issues/192#issuecomment-1403034902
> Thanks for testing. The API version doesn't get bumped when colorspaces are added. However, the library version will get bumped in the next release, either 3.0.5 or 3.1, and you can check against that.

@haasn: I agree with you, but see above. Current API version is 2.4, current lib version is 3.0.4. API version will not change, so I check for 3.0.5.